HISTORY.rst

```diff
@@ -49,6 +49,27 @@ Release History
 
 **IoT DPS updates**
 
+* Added RBAC support for DPS dataplane commands, similar to the RBAC support for IoT Hub.
```

Comment:
These DPS update notes need to be placed/combined with existing notes under 0.12.2 (unreleased). Then please change the 0.12.2 version in notes -> 0.13.0 and also make the same change to `VERSION` in `constants.py` from `0.12.2` to `0.13.0`.